content/docs/2.12/scalers/prometheus.md

```diff
@@ -32,6 +33,7 @@ triggers:
 
 - `serverAddress` - Address of Prometheus server. If using VictoriaMetrics cluster version, set full URL to Prometheus querying API, e.g. `http://<vmselect>:8481/select/0/prometheus`
 - `query` - Query to run.
+- `queryParameters` - Query Parameters to include while querying the prometheus endpoint.
```

Comment:
massive nit:
```suggestion
- `queryParameters` - A comma-seperated list of query parameters to include while querying the Prometheus endpoint.
```

Can you also align with https://github.com/kedacore/keda-docs#writing-documentation-for-a-scaler please?